src/gei/Commands/GenerateScriptCommand.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OctoshiftCLI.gei.Commands
```

Comment:
nit: `Gei` instead of `gei`

Reply:
I'm going to punt this to a follow-up PR where I overhaul the namespaces